tests/e2e/tslint.json

```diff
@@ -121,7 +116,8 @@
       "check-decl",
       "check-operator",
       "check-separator",
-      "check-type"
+      "check-type",
+      "check-module"
     ]
   }
 }
```

Comment:
Missing new line at the end of the file. Would be awesome to add a global linter rule for this

Reply:
thnx, added